File: auth-plugin/app/auth/controllers/AuthViews.java
package auth.controllers;

import auth.models.AuthenticatedUser;
import auth.models.BasicAuthViewResponse;
import com.google.inject.Inject;
import play.*;
import play.mvc.*;


public class AuthViews extends Controller {

    public Result signin() {
        BasicAuthViewResponse response = new BasicAuthViewResponse("Authentication", (AuthenticatedUser)ctx().args.get("auth-user"));
        return ok(auth.views.html.signin.render(response));
    }

}

File: auth-plugin/app/auth/controllers/AuthViews.scala
package auth.controllers

import auth.models.AuthenticatedUser
import common.models._
import play.api._
import play.api.mvc._


class AuthViews extends Controller {

    def signin() = Action {
        val response = new BasicViewResponse("Sign In")
        Ok(auth.views.html.signin.render(response))
    }

}
